src/dashboard/src/apache/django.wsgi

```diff
@@ -25,7 +27,20 @@ if path not in sys.path:
 
 os.environ['DJANGO_SETTINGS_MODULE'] = 'settings.common'
 
-application = get_wsgi_application()
+# workaround to troubleshoot mod_wsgi issues
+# See http://stackoverflow.com/questions/30954398/django-populate-isnt-reentrant 
+
+try:
+    application = get_wsgi_application()
+    print 'WSGI without exception'
+except Exception:
+    print 'handling WSGI exception'
+    # Error loading applications
+    if 'mod_wsgi' in sys.modules:
+        traceback.print_exc()
+        os.kill(os.getpid(), signal.SIGINT)
+        time.sleep(2.5)
+
```

Comment:
Should this re-raise the exception?  Some of the examples in https://groups.google.com/forum/#!topic/modwsgi/X3Upsk9Wfsw (linked from the SO question) have
```
except Exception:
    print('handling WSGI exception')
    # Error loading applications
    if 'mod_wsgi' in sys.modules:
        traceback.print_exc()
        os.kill(os.getpid(), signal.SIGINT)
        time.sleep(2.5)
   raise
```
The raise will probably kill that process, but I'm not sure of other ramifications.